src/plugins/emoteCloner/index.tsx

```diff
@@ -336,7 +339,31 @@ const messageContextMenuPatch: NavContextMenuPatchCallback = (children, props) =
                 const sticker = props.message.stickerItems.find(s => s.id === favoriteableId);
                 if (sticker?.format_type === 3 /* LOTTIE */) return;
 
-                return buildMenuItem("Sticker", () => fetchSticker(favoriteableId));
+                // Workaround for cases when it's not available
+                // (e.g when using MessageLinkEmkbeds)
+                if (sticker === undefined) return;
+
+                return buildMenuItem("Sticker", async () => {
+                    const fetchedSticker = await fetchSticker(favoriteableId);
```

Comment:
extract the fake sticker into a separate function and call it inline for nicer code

```suggestion
                return buildMenuItem("Sticker", () => fetchSticker(favoriteableId).then(s => s ?? buildFakeSticker(sticker));
```